jobs/transforms/collection_validation.py

```diff
@@ -20,28 +20,76 @@
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
 from core.domain import collection_domain
+from core.domain import rights_domain
 from core.platform import models
+from jobs import job_utils
 from jobs.decorators import validation_decorators
 from jobs.transforms import base_validation
 
 (collection_models,) = models.Registry.import_models([models.NAMES.collection])
 
 
 @validation_decorators.AuditsExisting(
-    collection_models.CollectionSnapshotMetadataModel,
-    collection_models.CollectionCommitLogEntryModel)
-class ValidateCollectionCommitCmdsSchema(
+    collection_models.CollectionSnapshotMetadataModel)
+class ValidateCollectionSnapshotMetadataModel(
         base_validation.BaseValidateCommitCmdsSchema):
     """Overrides _get_change_domain_class for collection models."""
 
     def _get_change_domain_class(self, input_model): # pylint: disable=unused-argument
-        """Returns a Change domain class.
+        """Returns a change domain class.
 
         Args:
             input_model: datastore_services.Model. Entity to validate.
 
         Returns:
-            change_domain.BaseChange. A domain object class for the
+            collection_domain.CollectionChange. A domain object class for the
             changes made by commit commands of the model.
         """
         return collection_domain.CollectionChange
+
+
+@validation_decorators.AuditsExisting(
+    collection_models.CollectionRightsSnapshotMetadataModel)
+class ValidateCollectionRightsSnapshotMetadataModel(
+        base_validation.BaseValidateCommitCmdsSchema):
+    """Overrides _get_change_domain_class for collection models."""
```

Comment:
Please update the docstring according to the class name(here and elsewhere).

Reply:
Done